fares-data-build-tool/create-data
# feat: cfd-309 manage passenger types in operator settings

# Description

-   This PR allows us to add and delete new passenger types under operator global settings.

# Testing instructions

-   Go to `/viewPassengerTypes` page and try adding and deleting a new passenger type, ensure data validation works.

# Type of change
